lib/seattleflu/db/session.py

```diff
@@ -142,6 +142,16 @@ def fetch_row(self, sql: str, values: Union[Tuple, Mapping] = None) -> Any:
             return cursor.fetchone()
 
 
+    def fetch_all(self, sql: str, values: Union[Tuple, Mapping] = None) -> Any:
+        """
+        Fetches all rows from the results of the *sql* query. Useful for
+        writing alerts for new rows in queries of interest.
+        """
+        with self.cursor() as cursor:
+            cursor.execute(sql, values)
+            return cursor.fetchall()
+
+
```

Comment:
I just realized that we are already using "fetch all" in other ETL processes (e.g. `etl presence-absence`) with the `cursor.execute` syntax. I will delete this commit.